src/main/java/seedu/address/logic/parser/LogCommandParser.java

```diff
@@ -25,7 +26,14 @@
     public LogCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_FLAG, PREFIX_NAME, PREFIX_HOURS);
-        Name studySpot = null;
+
+        // If command has no n/ and it's not a reset all command
+        if ((!arePrefixesPresent(argMultimap, PREFIX_NAME)
```

Comment:
Refer to my comment as line 60. This would be the default case and an exception is thrown. `arePrefixesPresent` would not be needed as you can make use of `isNamePresent` since we are only checking for name